x-pack/plugins/observability_solution/apm/server/routes/assistant_functions/get_observability_alert_details_context/get_downstream_dependency_name.ts

```diff
@@ -50,9 +49,10 @@ export async function getDownstreamServiceResource({
           ],
         },
       },
+      fields: ['span.destination.service'],
```

Comment:
this should be `span.destination.service*`?

Reply:
Thanks! We only use `span.destination.service.resource` so I changed it 👍